### PR TITLE
Critical security issue fix: executeTransactionFromOutside validation…

### PR DESCRIPTION
**Pull request by asimaranov**
PR fixes critical security issue, executeTransactionFromOutside doesn't actually check transaction verification result.
Many thanks for Vlad B for finding

### CODE REVIEW

File: docs/build/tutorials/smart-contract-development/account-abstraction/custom-aa-tutorial.md [hunk 1]
```diff
@@ -636,7 +638,9 @@ contract TwoUserMultisig is IAccount, IERC1271 {
         external
         payable
     {
-        _validateTransaction(bytes32(0), _transaction);
+        bytes4 magic = _validateTransaction(bytes32(0), _transaction);
+        require(magic == ACCOUNT_VALIDATION_SUCCESS_MAGIC, "NOT VALIDATED");
+
```
Comment (MexicanAce) [inline docs/build/tutorials/smart-contract-development/account-abstraction/custom-aa-tutorial.md:643]: nit:
```suggestion

```

File: docs/build/tutorials/smart-contract-development/account-abstraction/daily-spend-limit.md [hunk 2]
```diff
@@ -600,7 +600,9 @@ contract Account is IAccount, IERC1271, SpendLimit {
     function executeTransactionFromOutside(
         Transaction calldata _transaction
     ) external payable {
-        _validateTransaction(bytes32(0), _transaction);
+        bytes4 magic = _validateTransaction(bytes32(0), _transaction);
+        require(magic == ACCOUNT_VALIDATION_SUCCESS_MAGIC, "NOT VALIDATED");
+
```
Comment (MexicanAce) [inline docs/build/tutorials/smart-contract-development/account-abstraction/daily-spend-limit.md:605]: nit:
```suggestion

```

**Review by MexicanAce — APPROVED** [review]
LGTM

**Comment by asimaranov** [comment]
Also please consider to fix https://github.com/matter-labs/hardhat-zksync/pull/958

**Pull request merged**